Header header
int32 frame_id
BoundingBox[] detections